From 9c89f58135d18d7932283bf39bc43d47a85e720f Mon Sep 17 00:00:00 2001
From: Fugang Duan <B38611@freescale.com>
Date: Thu, 5 Apr 2012 17:15:30 +0800
Subject: ENGR00160472 - MX6: add Ethernet ANSI/IEEE 802.2 LLC support in
 defconfig.

- Add Ethernet ANSI/IEEE 802.2 LLC support. And the packet with
  IP head "ETH_P_802_2" will be processed in Ethernet stack L3 layer.
- If disable the feature, ethernet stack will drop the LLC packets.

Signed-off-by: Fugang Duan  <B38611@freescale.com>
---
 arch/arm/configs/imx6_defconfig |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index 601d450..eb2e332 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -563,7 +563,8 @@ CONFIG_DEFAULT_TCP_CONG="cubic"
 # CONFIG_NET_DSA is not set
 # CONFIG_VLAN_8021Q is not set
 # CONFIG_DECNET is not set
-# CONFIG_LLC2 is not set
+CONFIG_LLC=y
+CONFIG_LLC2=y
 # CONFIG_IPX is not set
 # CONFIG_ATALK is not set
 # CONFIG_X25 is not set
-- 
1.7.9.5

